backend/app/services/knowledge/knowledge_service.py

```
@@ -503,16 +512,14 @@ def list_knowledge_bases(
             # Single query to get personal and explicitly shared knowledge bases
             # Personal: user_id matches and namespace is "default"
             # Shared: id is in shared_kb_ids
-            all_kbs = (
-                db.query(Kind)
-                .filter(
+            all_kbs = exclude_code_wikis(
+                db.query(Kind).filter(
                     Kind.kind == "KnowledgeBase",
                     Kind.is_active == True,
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)